atomsci/ddm/pipeline/compare_models.py

```diff
@@ -1072,9 +1068,10 @@ def get_filesystem_models(result_dir, pred_type):
     """
     perf_df = get_filesystem_perf_results(result_dir, pred_type)
     if pred_type == 'regression':
-        metric = 'valid_r2_score'
+        _metric = 'valid_r2_score'
     else:
-        metric = 'valid_roc_auc_score'
+        _metric = 'valid_roc_auc_score'
```

Comment:
This will remain `metric` and will be resolved by adding logging so that L1075 (commented out code) can still work